script/hardhat/fork/vip-102.ts

```diff
@@ -0,0 +1,112 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+import { Comptroller } from "../../../typechain";
+import { forking, pretendExecutingVip, testVip } from "./vip-framework";
+import { ProposalType } from "./vip-framework/types";
+import { makeProposal } from "./vip-framework/utils";
+
+const COMPTROLLER = "0xfd36e2c2a6789db23113685031d7f16329158384";
+const vBUSD = "0x95c78222B3D6e262426483D42CfA53685A67Ab9D";
+
+const Actions = {
+  BORROW: 2,
+  ENTER_MARKETS: 7,
+};
+
+const vip102 = () => {
+  const meta = {
+    version: "v1",
```

Comment:
Should be v2, it's the version of the JSON schema for proposals

```suggestion
    version: "v2",
```

Reply:
Fixed